CMSgov/mint-app
# [EASI-2792] Search Change History GQL Result Typing

# EASI-2792

## Changes and Description

- Created a new type structure to contain search results



## How to test this change

Run any of the search endpoints. Verify that they work and return the changed fields as an array.

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
